pkg/doc/verifiable/credential_test.go

```diff
@@ -1639,3 +1639,40 @@ func TestNewUnverifiedCredential(t *testing.T) {
 		require.Nil(t, vc)
 	})
 }
+
+func TestMarshalCredential(t *testing.T) {
+	t.Run("test marshalling VC to JSON bytes", func(t *testing.T) {
+		vc, vcData, err := NewCredential([]byte(validCredential))
+		require.NoError(t, err)
+		require.NotNil(t, vc)
+		require.NotEmpty(t, vcData)
+
+		getvcMap := func(t *testing.T, vc *Credential) map[string]interface{} {
```

Comment:
You can use `toMap()` function from this package instead of `getvcMap()`

Reply:
done